src/audits/PageWithBadTitle.js

```diff
@@ -0,0 +1,43 @@
+// Copyright 2015 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+goog.require('axs.AuditRules');
+goog.require('axs.constants.Severity');
+
+axs.AuditRules.addRule({
+    name: 'pageWithBadTitle',
+    heading: 'The web page should have a title less than 66 characters long and does not contain bad characters',
```

Comment:
This suggested rule is to help users that depend on screen readers to browse the web;
Comes from a list of minor issues, collected by an accessibility expert who uses a screen reader on a daily basis;
I do not have any demographics research done on such a thing though;
I left the severity to a warning.